shared/css/top/top.css

```diff
@@ -38,11 +38,61 @@ body {
   word-wrap: break-word;
 }
 
+#post_content {
+    background: white;
+    border-radius: 6px;
+    padding: 13px;
+    box-shadow: 4px 5px 0px #8E8980;
+}
+
+#post_content p {
+  margin: 0;
+  word-wrap: break-word;
+}
+
+#cleartext {
+  height: 100%;
+}
+
+@media(min-width: 992px){
+  #dashed_hr {
```

Comment:
Uh yeah I wasn't sure how to trigger it, so I found that rule in bootstrap. 
```css
@media(min-width: 992px){
```
It's the exact same rule that bootstrap uses to when converting to vertical stacked, so there shouldn't be any problems other than it not being the cleanest approach.